pkg/utils/kubernetes/kubernetes_test.go

```diff
@@ -1187,7 +1187,7 @@ var _ = Describe("kubernetes", func() {
 			client.ObjectKey{Namespace: "", Name: "foo"},
 		),
 	)
-
+	// spellchecker:off
```

Comment:
Why not start right before the `certPEM` here but already several lines above?

Reply:
Done